Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -444,43 +424,44 @@ private float minX
         {
             get
             {
-                int hasMod = (ModuloCheck(NodeList.Count, Tiers) != 0) ? 1 : 0;
-                return NodeList.Count != 0 ? -((StepMultiplier(NodeList.Count - (ViewableArea * Tiers), Tiers) + hasMod) * CellWidth) : 0.0f;
+                int remainder = (SafeModulo(NodeList.Count, ItemsPerTier) != 0) ? 1 : 0;
+                return NodeList.Count != 0 ? -((SafeDivision(NodeList.Count - (ViewableArea * ItemsPerTier), ItemsPerTier) + remainder) * CellWidth) : 0.0f;
             }
-
         }
 
-        // Item index for items that should be visible
-        private int numItemsPrevView
+        /// <summary>
+        // Index of the first visible item in the nodes list
+        /// </summary>
+        public int FirstVisibleItemIndex
         {
             get
             {
                 if (scrollDirection == ScrollDirectionType.UpAndDown)
                 {
-                    return (int)Mathf.Ceil(scrollContainer.transform.localPosition.y / CellHeight) * Tiers;
+                    return (int)Mathf.Ceil(scrollContainer.transform.localPosition.y / CellHeight) * ItemsPerTier;
                 }
                 else
                 {
-                    return -((int)Mathf.Ceil(scrollContainer.transform.localPosition.x / CellWidth) * Tiers);
+                    return ((int)Mathf.Ceil(Mathf.Abs(scrollContainer.transform.localPosition.x / CellWidth)) * ItemsPerTier);
```

Comment:
Can you leave a comment on why (either here or in the code) the Abs() is required in the X case but not the Y case?